src/Features/Core/Portable/Completion/Providers/AbstractSymbolCompletionProvider.cs

```diff
@@ -80,20 +85,27 @@ private bool ShouldIncludeInTargetTypedCompletionList(ISymbol symbol, ImmutableA
                 }
             }
 
-            var type = symbol.GetMemberType() ?? symbol.GetSymbolType();
+            var type = (symbol.GetMemberType() ?? symbol.GetSymbolType()).WithoutNullability();
```

Comment:
@jasonmalinowski specifically for the `.WithoutNullability()` bit.

Reply:
Comment added where I suspect you need the actual fix.